cli/tests/cuda/cuda-disabled.sh

```diff
@@ -5,18 +5,7 @@ LDCONFIG_MOCK="${2}"
 
 # Get the function without loading support.
 _FLOX_ENV_CUDA_DETECTION=0 source "${FLOX_ENV}/etc/profile.d/0800_cuda.sh"
-
```

Comment:
The change in both of these `cuda-*.sh` scripts is necessary because `.flox/lib` is no longer added in `0800_cuda.sh`. This means you can't source that script to manually change `FLOX_ENV_LIB_DIRS`, and it no longer makes sense to detect the change here.

Furthermore, `.flox/lib` is now _always_ added to `FLOX_ENV_LIB_DIRS`, but I don't see how that would cause any problems.